build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+def hadoopAwsVersion = System.getenv('HADOOP_AWS_VER')
+def deltaSparkVersion = System.getenv('DELTA_SPARK_VER')
+def scalaVersion = System.getenv('SCALA_VER')
+
+dependencies {
+    implementation "org.apache.hadoop:hadoop-aws:$hadoopAwsVersion"
+    implementation "io.delta:delta-spark_${scalaVersion}:$deltaSparkVersion"
+    implementation "io.delta:delta-storage:$deltaSparkVersion"
```

Comment:
This is a dependency of delta-spark so not needed

Reply:
👍

Reply:
:+1: